### PR TITLE
[3044] - Display distance to nearest site and nearest address

### PR DESCRIPTION
**Pull request by TobyRet**
### Context
When querying courses by location we want course results listings to display the distance to the nearest site and the address of the nearest site.

When displaying the nearest distance the distance is rounded to 0 decimal places if it's less than 1 mile, else it is rounded to 1 decimal place.

### Changes proposed in this pull request
- Nearest site and site distance is displayed of user searches by location
- Main address is not displayed if user searches by location

distance_2

### CODE REVIEW

**Comment by TobyRet** [comment]
[[M] Make Find request location filtering](https://trello.com/c/SNDaCBWV/3044-m-make-find-request-location-filtering)

File: app/view_objects/results_view.rb [hunk 1]
```diff
@@ -135,6 +136,20 @@ def course_count
...
   end
 
+  def site_distance(course)
+    query_location = Geokit::LatLng.new(latitude.to_f, longitude.to_f)
+
+    site_coordinates = course.sites do |site|
+      { latitude: site.latitude, longitude: site.longitude }
+    end
+
+    distances = site_coordinates.map do |coordinates|
```
Comment (dankmitchell) [inline app/view_objects/results_view.rb]:
```suggestion
    distances = course.sites.map do |coordinates|
```

File: app/view_objects/results_view.rb [hunk 2]
```diff
@@ -135,6 +136,20 @@ def course_count
     courses.meta["count"]
   end
 
+  def site_distance(course)
+    query_location = Geokit::LatLng.new(latitude.to_f, longitude.to_f)
+
+    site_coordinates = course.sites do |site|
```
Comment (dankmitchell) [inline app/view_objects/results_view.rb]: Do we need this? based on the below suggestion...

Comment (aliuk2012) [reply app/view_objects/results_view.rb]: I agree, I don't think we need this.

**Comment by TobyRet** [comment]
PR updated to round site distance to 0 decimal places if site distance is greater than 1 mile, else it's rounded to 1 decimal place

**Pull request merged**